tools/docker/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+run() {
+    echo "$*"
+    "$@" || exit $?
+}
+
+scriptdir="$(cd "${0%/*}"; pwd)"
+topdir="${scriptdir%/*/*/*}"
+
+main() {
+    run docker container exec -it ${1-agent} ${topdir}/build/install/scripts/prplmesh_utils.sh status
```

Comment:
It would be better to use the -n option for the name, to make it similar to the run.sh.